README.md

```
@@ -1 +1,43 @@
-test branch cbhieu
\ No newline at end of file
+## Ứng dụng quản lý hệ thống cửa thông minh
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)